locales/fr/LC_MESSAGES/base.po

```diff
@@ -22,87 +22,140 @@ msgstr ""
 #: bot.py:58
 msgid "ECO_DESCRIPTION"
 msgstr ""
+"Eco n'est pas une banque. Eco est un solde unique qui vous donne jusqu'à 5% de APY sur vos dépôts et 5% de cashback chez un nombre croissant de commerçants."
+"En plus, bientôt, vous pourrez payer vos factures et envoyer de l'argent à vos amis via Eco. C'est un endroit simple et rémunérateur pour gérer toute votre vie financière."
+
 
 #: bot.py:64
 msgid "ECO_FIELD1_NAME"
-msgstr ""
+msgstr "A propos du projet"
 
 #: bot.py:65
 msgid "ECO_FIELD1_VALUE"
 msgstr ""
+"**Investisseurs:** a16z crypto, Founders Fund, Activant Capital, Slow Ventures, "
+"Coinbase Ventures, Tribe Capital, Valor Capital Group et autres\n"
+"**Fonds levés:** Plus de $35,000,000\n"
+"**Fonctionnalités de la plateforme actuellement utilisées par l'équipe:** paiements de factures, autres fonctionnalités intégrées à l'application, extension aux marchés internationaux"
```

Comment:
I propose `worked by the team` to be 'en cours d'implementation' ou `en cours de developement`